pkg/config/parser_test.go

```diff
@@ -30,131 +30,10 @@ provider "aws" {
   resources = ["slow_resource"]
 }`
 
-const testNoSource = `cloudquery {
```

Comment:
Moved these to files for:
1. Syntax highlighting
2. Formatting
3. Syntax validation